script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -102,7 +102,7 @@ All I want right now is to be left alone in my room. I want to take off my glass
 [sha:0c37400a99e466990576df7f2ed954e3895d4bdc]{
 -- Page 199, Offset 4478.
 -- @g　―――その女と、すれ違うまでは。
-@g ―――That is, until I crossed paths with that woman.
+@g ―――That is, until I crossed paths with that %{i}woman%{/i}.
```

Comment:
Don't think there's a need to italicize this.

Reply:
Does the `@g` not currently italicize this whole line? Same goes for the other italics inside `@g` blocks. Once the italics fix is in this scene will need some attention since it's got some heavy text effects, and maybe we'll want to go back to `@g` being a different font-face instead of an italic effect.